package.json

```diff
@@ -1,81 +1,75 @@
 {
-  "name": "@skyux/pages",
-  "version": "5.0.0-beta.3",
-  "description": "SKY UX Pages",
+  "name": "pages-workspace",
+  "version": "0.0.0",
   "scripts": {
-    "build": "skyux build-public-library --fullTemplateTypeCheck",
-    "skyux": "skyux",
-    "start": "skyux serve --hmr",
-    "test": "skyux test --coverage library --logFormat none",
-    "test:visual": "skyux e2e --logFormat none"
+    "ng": "ng",
+    "start": "ng serve",
+    "build": "ng build",
+    "watch": "ng build --watch --configuration development",
+    "test": "ng test",
+    "skyux:generate-lib-resources-module": "ng generate @skyux/i18n:lib-resources-module",
+    "lint": "ng lint",
+    "skyux:generate-documentation": "ng generate @skyux-sdk/documentation-schematics:documentation",
+    "postbuild": "npm run skyux:generate-documentation",
+    "e2e": "ng e2e"
   },
-  "keywords": [],
-  "author": "Blackbaud",
-  "license": "ISC",
-  "repository": {
-    "type": "git",
-    "url": "git+https://github.com/blackbaud/skyux-pages.git"
+  "private": true,
+  "dependencies": {
+    "@angular/animations": "~12.2.5",
+    "@angular/common": "~12.2.5",
+    "@angular/compiler": "~12.2.5",
+    "@angular/core": "^12.2.5",
+    "@angular/forms": "~12.2.5",
+    "@angular/platform-browser": "~12.2.5",
+    "@angular/platform-browser-dynamic": "~12.2.5",
+    "@angular/router": "~12.2.5",
+    "@skyux/assets": "^5.0.0-beta.2",
+    "@skyux/config": "^5.0.0-beta.2",
+    "@skyux/core": "^5.0.0-beta.11",
+    "@skyux/e2e-client": "^5.0.0",
+    "@skyux/forms": "^5.0.0-beta.5",
+    "@skyux/i18n": "^5.0.0-beta.7",
+    "@skyux/indicators": "^5.0.0-beta.4",
+    "@skyux/layout": "5.0.0-beta.7",
+    "@skyux/modals": "^5.0.0-beta.5",
+    "@skyux/packages": "^5.0.0-beta.96",
+    "@skyux/router": "^5.0.0-beta.2",
+    "@skyux/theme": "^5.0.0-beta.5",
+    "rxjs": "~6.6.0",
+    "tslib": "^2.3.0",
+    "zone.js": "~0.11.4"
   },
-  "bugs": {
-    "url": "https://github.com/blackbaud/skyux-pages/issues"
-  },
-  "homepage": "https://github.com/blackbaud/skyux-pages#readme",
   "peerDependencies": {
```

Comment:
The workspace `package.json` is different than what it was for libraries in SKY UX 4. The workspace `package.json` has nothing to do with the library `package.json` (and won't get published with NPM), so it doesn't require peer dependencies to be listed. Peer dependencies _should_ be defined for the `projects/pages/package.json`, however (which you've done).